.eslintrc

```diff
@@ -45,4 +44,4 @@
     "no-eval": "error",
     "no-unexpected-multiline": "off"
   }
-}
+}
```

Comment:
missing return at EOF

Reply:
IDE reverted to basic settings & autofixed all my files when I removed the prettier dependency. Thanks for the heads-up :+1: